docs/root/version_history/current.rst

```diff
@@ -25,6 +25,7 @@ Removed Config or Runtime
 * http: removed ``envoy.reloadable_features.require_strict_1xx_and_204_response_headers`` and ``envoy.reloadable_features.send_strict_1xx_and_204_response_headers`` and legacy code paths.
 * http: removed ``envoy.reloadable_features.strip_port_from_connect`` and legacy code paths.
 
+* upstream: removed ``envoy.reloadable_features.health_check.graceful_goaway_handling`` and legacy code paths.
```

Comment:
Can you merge main and remove the newline above this line? Thank you.

Reply:
Done.